fastly/resource_fastly_service_v1_s3logging_test.go

```diff
@@ -78,6 +78,46 @@ func TestAccFastlyServiceV1_s3logging_basic(t *testing.T) {
 	})
 }
 
+func TestAccFastlyServiceV1_s3logging_domain_default(t *testing.T) {
+	var service gofastly.ServiceDetail
+	name := fmt.Sprintf("tf-test-%s", acctest.RandString(10))
+	domainName1 := fmt.Sprintf("%s.notadomain.com", acctest.RandString(10))
+
+	log1 := gofastly.S3{
+		Version:           1,
+		Name:              "somebucketlog",
+		BucketName:        "fastlytestlogging",
+		Domain:            "s3.amazonaws.com",
+		AccessKey:         "somekey",
+		SecretKey:         "somesecret",
+		Period:            uint(3600),
+		GzipLevel:         uint(0),
+		Format:            "%h %l %u %t %r %>s",
+		FormatVersion:     1,
+		TimestampFormat:   "%Y-%m-%dT%H:%M:%S.000",
+		ResponseCondition: "response_condition_test",
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckServiceV1Destroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccServiceV1S3LoggingConfig_domain_default(name, domainName1),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckServiceV1Exists("fastly_service_v1.foo", &service),
+					testAccCheckFastlyServiceV1S3LoggingAttributes(&service, []*gofastly.S3{&log1}),
+					resource.TestCheckResourceAttr(
+						"fastly_service_v1.foo", "name", name),
+					resource.TestCheckResourceAttr(
+						"fastly_service_v1.foo", "s3logging.#", "1"),
+				),
```

Comment:
I guess we could also check that new default value of `s3logging.<idx>.domain` here, but it's no big deal.